CONTRIBUTING.md

```diff
@@ -95,7 +95,10 @@ python convert_to_tf_keras.py --revert
 
 Note that you are strongly encouraged to commit your code before in case the parsing would go wrong. To discard all the changes you made since the previous commit:
 ```
-git reset --hard
+# saves a copy of your current codebase in the git stash and comes back to the previous commit
+git reset && git stash
```

Comment:
Using `git stash` is enough, you don't need the `git reset`  [doc](https://www.git-scm.com/docs/git-stash)

Reply:
Done